Sample Solutions/YUME/Yume_Connector_MVP_vsol-yume-connector.swagger.json

```diff
@@ -0,0 +1,335 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "vsol-graph-analytics-connector",
+    "description": "Custom connector to microsoft graph, analytics, site and profile for the yme powerapp",
```

Comment:
"Yume"

Reply:
Updated this and the Readme.md based on your comments